### PR TITLE
Filter order fixes

### PR DESCRIPTION
**Pull request by pannkotsky**

### CODE REVIEW

**Comment by barseghyanartur** [comment]
@pannkotsky:

Hey, thanks. I'll check this ASAP.

File: src/graphene_elastic/fields.py [hunk 1]
```diff
@@ -235,13 +235,13 @@ def get_queryset(self, document, info, **args):
                     hydrated_references[arg_name] = reference_obj
             args.update(hydrated_references)
 
+        qs = document.search()
         if self._get_queryset:
             queryset_or_filters = self._get_queryset(document, info, **args)
             if isinstance(queryset_or_filters, elasticsearch_dsl.Search):
-                return queryset_or_filters
```
Comment (barseghyanartur) [inline src/graphene_elastic/fields.py:241]: This does change the default behavior.

File: src/graphene_elastic/fields.py [hunk 2]
```diff
@@ -235,13 +235,13 @@ def get_queryset(self, document, info, **args):
...
+                qs = queryset_or_filters
```
Comment (barseghyanartur) [inline src/graphene_elastic/fields.py:242]: This does change the default behavior.

File: src/graphene_elastic/fields.py [hunk 3]
```diff
@@ -235,13 +235,13 @@ def get_queryset(self, document, info, **args):
                     hydrated_references[arg_name] = reference_obj
             args.update(hydrated_references)
 
+        qs = document.search()
```
Comment (barseghyanartur) [inline src/graphene_elastic/fields.py:238]: Could you explain the need of this?

**Review by barseghyanartur — COMMENTED** [review]
Sorry for huge delay. See my remarks.

**Comment by pannkotsky** [comment]
Sorry, I completely forgot about this PR and can't really remember anything about it anymore

**Pull request closed without merging**